src/migration-wizard/steps/discovery/EnhancedDownloadButton.tsx

```
@@ -0,0 +1,654 @@
+      pdf.save('Dashboard_Report.pdf');
+
+      hiddenContainer.innerHTML = '';
+      console.warn = originalWarn;
+    } catch (error) {
+      console.error('Error generating PDF:', error);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+const handleHTMLExport = async () => {
```

ESLint

[error] Insert `··`

(prettier/prettier)